cpu/nrf52/Makefile.include

```diff
@@ -1,5 +1,11 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = nrf52
 
+# Export internal ROM alignment and slot sizes for bootloader support
+export IMAGE_ALIGN = 8
+export SLOT0_SIZE = 0x8000
+export SLOT1_SIZE = 0x3C000
+export SLOT2_SIZE = 0x3C000
```

Comment:
Agree with @kaspar030 (and that is what I had in mind in some earlier comments somewhere), but let's merge this for now and clean this up later.

By the way, this unconditional slot size definition is also calling for trouble as there are nrf52s with differing ROM size out there... But lets fix later.